### PR TITLE
DOC: Adds Iñigo's week 12 blogpost

### PR DESCRIPTION
**Pull request by itellaetxe**
Adds my last blog post of GSoC 2024

### CODE REVIEW

**Comment by itellaetxe** [comment]
I attempted to fix the bullet point list renderization because the first item of the first list was being added inline. I think the attempts were successful but my browser was storing the first version of the page in the cache or something, and I was not able to see the updates. I tried in incognito mode and it worked. FYI...

**Review by jhlegarreta — COMMENTED** [review]
Please squash all commits into a single commit.

File: posts/2024/2024_08_16_Inigo_week_12.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
+Week 12 into GSoC 2024: Last weeks of the coding phase and admin stuff
```
Comment (WassCodeur) [inline posts/2024/2024_08_16_Inigo_week_12.rst:1]: Hi @itellaetxe!

Nice blog. I have just one suggestion to make, see below

File: posts/2024/2024_08_16_Inigo_week_12.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+Lastly, I also progressed a bit with the last post, writing a bit more about the work I already completed.
+
+What is coming up next week
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Next week I will:
+
+- Open the draft PR for the vanilla AutoEncoder class and tests.
+- Finish the final post, linking it to the draft PR. I will probably do this on the weekend so I can have a first version on Monday and thus have a couple of days to polish it.
+- Polish the final post with my mentors.
```
Comment (WassCodeur) [inline posts/2024/2024_08_16_Inigo_week_12.rst:37]: Perhaps you could quote your mentors.

**Review by WassCodeur — APPROVED** [review]
LGTM

File: posts/2024/2024_08_16_Inigo_week_12.rst (hunk 2 above)
Comment (itellaetxe) [reply posts/2024/2024_08_16_Inigo_week_12.rst:37]: Got it, thanks 🤝🏽

**Review by robinroy03 — APPROVED** [review]

**Review by deka27 — APPROVED** [review]

**Comment by skoudoro** [comment]
all ok for you @jhlegarreta and @pjsjongsung ? Can I go ahead and merge ?

**Review by jhlegarreta — APPROVED** [review]

**Pull request merged**